modules/oxide-components/package.json

```diff
@@ -26,7 +26,7 @@
   ],
   "main": "lib/main.js",
   "module": "lib/main.js",
```

Comment:
don't these need changing as well? 🤔

Reply:
No. Vite spits out the bundled file in this location. Unless you want us to change the location of where the bundled file is spit out ?

Reply:
All good I'm just used to `tsc`. It doesn't really matter as long as it's git-ignored.